src/index.html

```diff
@@ -11,7 +11,141 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    autocomplete="off">
+      <fieldset class="section section1">
+        <legend>Personal information</legend>
+        <div class="form__field">
+          <label>
+            Surname:
+            <input type="text" name="surname">
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            Name:
+            <input type="text" name="name">
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            How old are You?
+            <input type="number"
+              name="age"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            Full date of birth:
+            <input type="date" name="birth-date">
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            I accept the term of the agreement
+            <input type="checkbox" name="tos-box">
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="section section2">
+        <legend>Registration</legend>
+        <div class="form__field">
+          <label>
+            E-mail:
+            <input type="email"
+              name="email-addr"
+              placeholder="email@example.com"
+              required
+            >
```

Comment:
if multi-lining, each attribute should start at new line
```suggestion
            <input
              type="email"
              name="email-addr"
              placeholder="email@example.com"
              required
            >
```